src/NodaTime/NodaConstants.cs

```diff
@@ -159,12 +159,18 @@ public static class NodaConstants
         /// <summary>
         /// The instant at the Unix epoch of midnight 1st January 1970 UTC.
         /// </summary>
-        public static readonly Instant UnixEpoch = Instant.FromUnixTimeTicks(0);
+        public static Instant UnixEpoch { get; } = Instant.FromUnixTimeTicks(0);
 
         /// <summary>
         /// The instant at the BCL epoch of midnight 1st January 0001 UTC.
         /// </summary>
-        public static readonly Instant BclEpoch = Instant.FromUtc(1, 1, 1, 0, 0);
+        public static Instant BclEpoch { get; } = Instant.FromUtc(1, 1, 1, 0, 0);
+
+        /// <summary>
+        /// The epoch used for Julian dates. This is noon (UTC) on January 1st 4713 BCE in the proleptic
```

Comment:
"The instant at the Julian epoch of ...", for consistency with {Unix,Bcl}Epoch, maybe?